exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/CreateMissingTablesAndColumnsTests.kt

```diff
@@ -520,7 +520,8 @@ class CreateMissingTablesAndColumnsTests : DatabaseTestsBase() {
             val traceNumber = reference("traceNumber", ordersTable.traceNumber)
         }
 
-        withDb {
+        // Oracle metadata only returns foreign keys that reference primary keys
+        withDb(excludeSettings = listOf(TestDB.ORACLE)) {
```

Comment:
An alternative to excluding Oracle would be to change the mapping of the test tables so that a primary key is referenced instead of a unique index:
```kt
val ordersTable = object : Table("tmporders") {
    val traceNumber = char("traceNumber", 10)
    override val primaryKey = PrimaryKey(traceNumber)
}
val receiptsTable = object : IntIdTable("receipts") {
    val traceNumber = reference("traceNumber", ordersTable.traceNumber)
}
```
This would still retain [the original intention](https://github.com/JetBrains/Exposed/issues/1282) of the unit test, I think, which seems more about column naming than whether the column is a primary key or a unique index.